web-server/src/components/BranchSelector.tsx

```diff
@@ -252,14 +254,15 @@ export const BranchSelector: FC = () => {
           <Option>
             <OptionTitle
               selected={isAllMode}
-              onSelect={() =>
+              onSelect={() => {
                 dispatch(
                   appSlice.actions.setBranchState({
                     mode: ActiveBranchMode.ALL,
                     names: ''
                   })
-                )
-              }
+                );
+                depFn(localBranchNames.set, []);
```

Comment:
`localBranchNames.set` should be included in the dependency array of the `setProdStateBranchNames` callback